packages/fxa-customs-server/lib/actions.js

```diff
@@ -75,6 +75,11 @@ const RESET_PASSWORD_OTP_VERIFICATION_ACTION = {
   passwordForgotVerifyOtp: true,
 };
 
+// Actions that result in calls to the twilio api
```

Comment:
Doesn't this list include all/most sms actions? Is the intention here to reduce the chances of being rate limited by Twilio?